application/language/danish/kalkun_lang.php

```diff
@@ -326,8 +326,8 @@
 $lang['Access denied. Only administrators are allowed to manage plugins.'] = 'Only administrators can manage plugins';
 $lang['Plugin {0} installed successfully.'] = 'Plugin {0} successfully installed';
 $lang['Plugin {0} uninstalled successfully.'] = 'Plugin {0} successfully uninstalled';
-$lang[' - Installed'] = ' - Installed';
```

Comment:
How are `['repetition']`, `['Plural']` used ? What is the different between just using `$lang['repetition']` and this ?

Reply:
They are just here to help distinguish the context or help the translator understand how he should translate the word.

`tr('times', 'repetition')` will search for `$lang['times']['repetition']` and display its value.


Otherwise, a translator will not know if times has the meaning of "repetition" or of "time of the day"